tests/cases/exporting/tests.py

```diff
@@ -25,30 +47,56 @@ def setUp(self):
             'sort': 'desc',
         }])
 
-        proc = QueryProcessor(view=view, tree=models.Employee)
-        self.results = proc.get_iterable()
+        proc = QueryProcessor(view=view)
+        queryset = proc.get_queryset()
+        self.iterable = proc.get_iterable(queryset=queryset)
+        self.exporter = proc.get_exporter(export.BaseExporter)
 
-        # Ick..
-        self.exporter = export.BaseExporter(view)
-        self.exporter.params.insert(0, (RawFormatter(keys=['pk']), 1))
-        self.exporter.row_length += 1
+    def test_read(self):
+        it = self.exporter.read(self.iterable)
+        rows = list(it)
+        self.assertEqual([r[0] for r in rows], [2, 4, 6, 1, 3, 5])
```

Comment:
Consider creating a constant for `[2, 4, 6, 1, 3, 5]` as it is used repeatedly below and any change to the data will require updating all those lists or, just one constant.